bus_board_part2.rb

```diff
@@ -0,0 +1,100 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+module WebRequest
+  def web_request(uri)
+
+    response = Net::HTTP.get_response(uri)
+
+    JSON.parse(response.body)
+
+  end
+end
+
+class BusStop
+  include WebRequest
+
+  attr_accessor :stop_id, :buses_info
+
+  def initialize(stop_id)
+    @stop_id = stop_id
+    @buses_info = nil
+  end
+
+  def fetch_buses_info
+    uri = URI("https://api.tfl.gov.uk/StopPoint/#{@stop_id}/Arrivals")
+    @buses_info = web_request(uri)

```

Comment:
Every time `web_request` is called the line above it is creating a URI. That looks like repeated code to me! Can you find a way to stop you needing to do that?